### DISCUSSION TITLE
failed to load templated assets

### DISCUSSION THREAD

**Discussion by washCutBlow** [authelia/authelia#6703] [Q&A] [answered]
<img width="1346" alt="图片" src="https://github.com/authelia/authelia/assets/26592450/bbee80b3-e33d-444e-bb34-c775836944b2">

Can you help me ?

**Comment by glime-ai** [comment]
Please note this response was created using generative AI leveraging previously available information from the repository. This information may be helpful however it is our strong recommendation that you carefully consider the information before blindly applying it.

---

Based on the context provided, it appears you are seeking assistance with an image related to Authelia, as indicated by the URL structure in your question. However, the URL you've provided points to an image hosted on GitHub under the Authelia repository, but it doesn't directly lead to a specific issue, documentation, or any content that can be directly addressed without seeing the image itself.

To provide you with the most accurate assistance, could you please clarify the following:
1. What specific issue or question do you have regarding Authelia? Are you facing a configuration problem, an error message, or do you need guidance on how to use a particular feature?
2. If the image is related to an error or a configuration setup, could you describe what the image shows or provide the error message in text form?

This additional information will help in understanding your query better and offering a precise solution.

This comment was generated by [Glime](https://glimelab.ai/).

**Comment by james-d-elliott** [comment]
Explain your issue and what you're trying to do.

**Reply by washCutBlow** [reply]
I exec the command 

authelia-scripts build. on linux system . and generate the dist directry files

then i copy the files to mac os.  and  exec the below cmd

CGO_ENABLED=1 CGO_CPPFLAGS="-D_FORTIFY_SOURCE=2 -fstack-protector-strong" CGO_LDFLAGS="-Wl,-z,relro,-z,now" \
go build -ldflags "-linkmode=external" -trimpath -buildmode=pie -o authelia ./cmd/authelia

generate the binary file authelia . and move the authelia to the dist directry . and exec the file  it output the error that can't find the file

**Reply by james-d-elliott** [reply]
Yeah the file needs to exist. What are you trying to do? Why not use the compiled containers/binaries?

**Reply by washCutBlow** [reply]
but the file is already exist. as shown in my screenshot picture.   I want to run the system on my mac os for learn more detail

**Reply by james-d-elliott** [reply]
Your screenshot doesn't show the file.

**Reply by washCutBlow** [reply]
![Uploading 图片.png…]()

**Reply by washCutBlow** [reply]
<img width="1324" alt="图片" src="https://github.com/authelia/authelia/assets/26592450/7a16d6bd-be94-406b-a6ab-28019d70d392">

**Reply by james-d-elliott** [reply]
~~It's not trying to open any of those files....~~ Seems like that was not in the original screenshot. As I said. Amazing.

**Reply by james-d-elliott** [reply, marked as answer]
You'll need to put it in the embedded assets directory at the time you build it.

**Reply by washCutBlow** [reply]
embedded assets directory you mean which directory ?

**Reply by james-d-elliott** [reply]
internal/sever/public_html per the guide.
